cfgov/v1/util/ref.py

```diff
@@ -33,7 +33,7 @@
     ('amicus-brief', 'Amicus Brief'),
     ('blog', 'Blog'),
     ('enforcement', 'Enforcement Action'),
-    ('final-rule', 'Final Rule'),
+    ('final-rule', 'Final rule'),
```

Comment:
Maybe undo this one, since it's not public-facing and it would then be inconsistent with the rest of this list?

Reply:
Hmm I see what you're saying.  It's used when a content editor selects a page type, and you're right that they didn't request changes on the other ones, however given the request I'd rather just change it everywhere I can change it.

Reply:
This is how it looks now with the change. As you can see, there are already inconsistencies in terms of casing in the list.  I think I'm OK with leaving this as-is.
<img width="576" alt="screen shot 2018-02-13 at 2 26 28 pm" src="https://user-images.githubusercontent.com/354591/36177242-eca07ce6-10c9-11e8-91d2-2c21b79cedad.png">

Reply:
@richaagarwal - I think more of those can be changed to sentence case. I am checking with @sarahsimpson09 now.

Reply:
@nataliafitzgerald It'd be easier if that came in through a new request since updating this PR requires updating a migration, especially since that specific list is only internally-facing.

Reply:
(That said, I'm not really sure why Final rule was included in the first place... so maybe that's what's causing the confusion)

Reply:
Yes, if we want to go ahead and merge these changes and then I can update the original issue with any other changes that's fine. "Final rule" is externally facing so I would like to keep that in the set we're correcting. https://www.consumerfinance.gov/policy-compliance/rulemaking/final-rules/